deps/source_package/debian/control

```diff
@@ -0,0 +1,35 @@
+Source: edubuntu-meta
+Section: metapackages
+Priority: optional
+Maintainer: Deepanshu Gajbhiye <gajbhiye.deepanshu@gmail.com>
+Standards-Version: 4.1.4
+Build-Depends: debhelper (>= 11)
+ 
+Package: debian-edu-preschool
+Architecture: any
+Depends: calibre, chemtool, fritzing, inkscape, kalzium, kmplot, ktouch, kturtle, lightspeed, marble, melting, pencil2d, rocs, step, yorick
+Description: Preschool Educational Application Bundle
+ This package depends on all of the educational software for preschool aged 
+ children that is fully supported by Canonical and the Edubuntu community.
+
+Package: debian-edu-primary
+Architecture: any
+Depends: gcompris, kalgebra, kalzium, kanagram, kbruch, kgeography, khangman, kig, klettres, kmplot, kstars, ktouch, ktuberling, kturtle, kwordquiz, marble, parley, ri-li, step, tuxmath, tuxpaint, tuxtype
+Description: Primary Educational Application Bundle
+ This package depends on all of the educational software for Primary grade
+ level education that is fully supported.
+
+Package: debian-edu-secondary
+Architecture: any
+Depends: calibre, chemtool, einstein, fritzing, inkscape, kalgebra, kalzium, kbruch, kgeography, kig, kmplot, kstars, ktouch, ktuberling, kturtle, kwordquiz, lightspeed, marble, melting, parley, pencil2d, ri-li, step
+Description: Secondary Educational Application Bundle
+ This package depends on all of the educational software for Secondary grade
+ level education that is fully supported.
+
+Package: debian-edu-tertiary
+Architecture: any
+Depends: calibre, cantor, chemtool, fritzing, inkscape, kalzium, kmplot, ktouch, kturtle, lightspeed, marble, melting,pencil2d, rocs, step, yorick
+Description: Tertiary Educational Application Bundle
+ This package depends on all of the educational software for Tertiary grade
+ level education that is fully supported.
```

Comment:
You still need to fix the list of dependences for each package. For example the dependences of `debian-edu-preschool` are the same as those of `debian-edu-terciary`. This cannot be right. Why are you so careless?

Reply:
@d78ui98 compare dependencies here,

preschool: https://packages.ubuntu.com/bionic/ubuntu-edu-preschool
tertiary: https://packages.ubuntu.com/bionic/ubuntu-edu-tertiary

Reply:
Thanks.